### PR TITLE
[Docs] Plugins section update

### PR DESCRIPTION
**Pull request by CoderMaggie**
| Q               | A
| --------------- | -----
| Branch?         | 1.3
| License         | MIT

Our Plugins section required updating, especially finally listing the official plugins in the docs.

### CODE REVIEW

File: docs/plugins/accepted-plugins.rst [hunk 1]
```diff
@@ -0,0 +1,27 @@
+Plugins Accepted by Sylius
```
Comment (Zales0123) [inline docs/plugins/accepted-plugins.rst]: The badge is `Approved by Sylius` so I would keep this naming in the documentation as well :)

File: docs/plugins/accepted-plugins.rst [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+Since Sylius is an open-source platform, there is a certain flow in order for the plugin to become officially adopted by the community.
+
+**1.** Develop the plugin using :doc:`the official Plugin Development guide </plugins/creating-plugin>`.
+
+**2.** Remember about the tests and code quality!
+
+**3.** Send it to the project maintainers. It can be via email to any member of the Sylius Core team, or `the official Sylius Slack <http://sylius.com/slack>`_.
+
+**4.** Wait for your Plugin to be featured in `the list of plugins <http://sylius.com/developers/store/plugins>`_ on the Sylius website.
```
Comment (Zales0123) [inline docs/plugins/accepted-plugins.rst]: Maybe we should add another point before this one, something like `One of our Plugin Curators will contact you with the feedback regarding your plugin's code quality, test suite, and a general feeling. Will also ask you to provide some changes in the code to make this plugin approved.`

File: docs/plugins/index.rst [hunk 3]
```diff
@@ -19,23 +13,16 @@ are built on top of the Sylius Core.
...
-
-**1.** Develop the plugin using :doc:`the official Plugin Development guide </plugins/creating-plugin>`.
-
-**2.** Remember about the tests and code quality!
-
-**3.** Send it to the project maintainers. It can be via email to any member of the Sylius Core team, or `the official Sylius Slack <http://sylius.com/slack>`_.
+.. tip::
 
-**4.** Wait for your Plugin to be featured `the list of plugins <http://sylius.com/developers/store/plugins>`_ on the Sylius website.
+    **The list of all Sylius Plugins (offcial and accepted) is available on the Sylius website** `here <http://sylius.com/developers/store/plugins>`_.
```
Comment (Zales0123) [inline docs/plugins/index.rst]: `offcial and accepted` -> `official and approved`

File: docs/plugins/approved-plugins.rst [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+Since Sylius is an open-source platform, there is a certain flow in order for the plugin to become officially adopted by the community.
+
+**1.** Develop the plugin using :doc:`the official Plugin Development guide </plugins/creating-plugin>`.
+
+**2.** Remember about the tests and code quality!
+
+**3.** Send it to the project maintainers. It can be via email to any member of the Sylius Core team, or `the official Sylius Slack <http://sylius.com/slack>`_.
+
+**4.** One of our Plugin Curators will contact you with the feedback regarding your plugin's code quality, test suite,
```
Comment (lchrusciel) [inline docs/plugins/approved-plugins.rst]: I'm thinking if it makes sense to list our Plugin Curators, or link to Holaspirit directly: https://app.holaspirit.com/public/sylius/circle/YzgQ1c-sylius-ecommerce-framework/role/fDFHpy-plugin-curator WDYT?

**Comment by lchrusciel** [comment]
Thank you, Magdalena! :1st_place_medal:

**Pull request merged**